jellyfin-launcher.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+PORT=8096
+
+if ! curl --silent http://localhost:$PORT > /dev/null; then
+  flatpak run org.jellyfin.Jellyfin & sleep 10
```

Comment:
Since this script is the `command`, this is completely wrong. You don't `flatpak run` inside flatpak.

Reply:
Updated script